nolearn/lasagne/tests/test_base.py

```diff
@@ -114,6 +114,22 @@ def test_save_params_to_path(self, net, net_fitted, X_test, y_pred):
         net_loaded.load_params_from(path)
         assert np.array_equal(net_loaded.predict(X_test), y_pred)
 
+    def test_load_params_from_message(self, net, net_fitted, X_test, y_pred,
+                                      capsys):
+        net_loaded = clone(net)
```

Comment:
I think I used `net_loaded` in tests for nets that were loaded from disk.  Maybe `net` alone is fine.